src/locales/fr/v2/getting-started.adoc

```diff
@@ -281,15 +282,15 @@ Voici la réponse
 
 . Il manque une parenthèse à la fonction `setTempo()`.
 . Il manque un « e » à la première fonction `fitMedia()`.
-. Il manque une virgule entre le troisième et le quatrième paramètre de la première fonction `fitMedia()`.
+. Il manque une virgule entre le troisième et le quatrième argument de la première fonction `fitMedia()`.
```

Comment:
@leaikkache In the English version, we have standardized on "parameters" instead of arguments here.

Reply:
Hi,
From what I understand, arguments are for when you call the function, and parameters are for when you define the function.

Reply:
That is accurate, but we do not introduce the term "argument" at all in this chapter, and "argument" is not used in any of the English versions of these v2 files.

Ideally, I think the translations should reflect the original English text and not change terms or introduce new terms. Thoughts?

Reply:
My personal opinion is that it's better to be more accurate. In the teaching materials I created in French, the terms are all defined. I suggested a change in the English version as well (and the English teaching materials also take into accoun the difference). 
However I understand if you think it's too complicated to make the changes. I think it would be best if you looked at all the pull requests at the same time to have a good overall understanding of the changes (and some of them are only missing translations so it'd be nice to have them online asap).